libcxx/src/atomic.cpp

```diff
@@ -69,17 +69,17 @@ extern "C" int __ulock_wait(
     uint32_t operation, void* addr, uint64_t value, uint32_t timeout); /* timeout is specified in microseconds */
 extern "C" int __ulock_wake(uint32_t operation, void* addr, uint64_t wake_value);
 
```

Comment:
It's probably worth adding a `static_assert(sizeof(__cxx_atomic_contention_t) == 8);`.